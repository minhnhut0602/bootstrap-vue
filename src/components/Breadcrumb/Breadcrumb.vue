<template>
    <nav>
        <ol class="breadcrumb">
            <slot />
        </ol>
    </nav>
</template>
