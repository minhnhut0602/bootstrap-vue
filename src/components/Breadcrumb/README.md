# Breadcrumb

```html
<Breadcrumb>
    <BreadcrumItem>
        Home
    </BreadcrumItem>

    <BreadcrumItem active>
        Products
    <BreadcrumItem>
</Breadcrumb>
```

## Slots

| Name | Description |
| - | - |
| default | Displays content inside the breadcrumb. |
