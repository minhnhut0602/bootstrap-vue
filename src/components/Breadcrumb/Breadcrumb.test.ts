import Breadcrumb from '.';

describe('template', () => {
    componentRenderTest(Breadcrumb);

    componentSlotRenderTest(Breadcrumb);
});
