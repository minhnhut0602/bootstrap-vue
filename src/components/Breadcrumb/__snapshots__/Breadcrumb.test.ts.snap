// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template renders default 1`] = `
"<nav>
  <ol class=\\"breadcrumb\\">
    <!-- @slot Displays content inside the breadcrumb -->
  </ol>
</nav>"
`;
